.babelrc

```diff
@@ -0,0 +1,2 @@
+
+{     "presets": ["@babel/preset-env", "@babel/preset-react"] }
```

Comment:
Is this enough for us to be able to use es6 and async await?

Reply:
Yes@babel/preset-env helps us use the latest JavaScript and that can allow us to use es6.